docs/guide/cli.md

```diff
@@ -40,6 +40,15 @@ Useful to run with [`lint-staged`](https://github.com/okonet/lint-staged) or wit
 vitest related /src/index.ts /src/hello-world.js
 ```
 
+Please note that if you want to use it with `lint-staged`, you probably should also pass `--run` so that `lint-staged` command can exit normally.
```

Comment:
I'm not sure there's a need to document this because `vitest run` is already documented. @sheremet-va what do you think?

Reply:
Vitest detects CI automatically. I think #2705 should help, so there is no need to pass down `--run` manually.

Reply:
@sheremet-va It's not just about ci. `lint-staged` may be used as a git hook, which normally doesn't run in a ci.

Reply:
I updated the wording. @dammy001 can you review it?